<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YESport tracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>yesport_icon.png</normaloff>yesport_icon.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Відділення</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="departmentsList">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addDepartmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Додати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Пошук</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="quickSearchEdit">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addVisitorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Додати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="usersList"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>quickSearchEdit</tabstop>
  <tabstop>addVisitorButton</tabstop>
  <tabstop>usersList</tabstop>
  <tabstop>departmentsList</tabstop>
  <tabstop>addDepartmentButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
